File: src/main/java/com/xavierbouclet/whiskies/api/repository/WhiskyRepository.java
package com.xavierbouclet.whiskies.api.repository;

import com.xavierbouclet.whiskies.api.model.Whisky;
import org.springframework.data.repository.reactive.ReactiveCrudRepository;

import java.util.UUID;

public interface WhiskyRepository extends ReactiveCrudRepository<Whisky, UUID>{
}

File: src/main/kotlin/com/xavierbouclet/whiskies/api/repository/WhiskyRepository.kt
package com.xavierbouclet.whiskies.api.repository;

import com.xavierbouclet.whiskies.api.model.Whisky;
import org.springframework.data.repository.ListCrudRepository;
import org.springframework.data.repository.ListPagingAndSortingRepository;
import org.springframework.data.repository.PagingAndSortingRepository;

import java.util.UUID;

public interface WhiskyRepository extends ListPagingAndSortingRepository<Whisky, UUID>, ListCrudRepository<Whisky, UUID> {
}
